test/metabase/test/data/interface.clj

```diff
@@ -230,24 +230,36 @@
   actual instance, *not* the definition) of the Metabase object to return (e.g., a pass a `Table` to a
   `FieldDefintion`). For a `DatabaseDefinition`, pass the driver keyword."
   {:arglists '([db-or-table-or-field-def context])}
-  (fn [db-or-table-or-field-def context] (class db-or-table-or-field-def)))
+  (fn [db-or-table-or-field-def _context]
+    (class db-or-table-or-field-def)))
 
 (defmethod metabase-instance FieldDefinition
   [this table]
-  (Field :table_id (:id table), :%lower.name (str/lower-case (:field-name this))))
+  (db/select-one Field
+                 :table_id    (u/the-id table)
+                 :%lower.name (str/lower-case (:field-name this))
+                 {:order-by [[:id :asc]]}))
```

Comment:
The stuff here is all minor tweaks to add `ORDER BY id ASC` so the first matching object is returned. I ran into a weird situation running from the REPL where `(mt/id)` was indeterminate because I had two matching H2 `test-data` databases